### PR TITLE
fix: import votes

### PR DESCRIPTION
**Pull request by hussedev**
fixes PAR-343
fixes PAR-348

### CODE REVIEW

File: src/features/ballot/components/ImportCSV copy.tsx
Comment (nijoe1) [inline src/features/ballot/components/ImportCSV copy.tsx:1]: Do we need this ? Can you delete ?

File: src/features/ballot/components/ImportCSV.tsx [hunk 1]
```diff
@@ -20,23 +24,34 @@ export function ImportCSV({
...
+    }
   }, []);
 
   return (
     <>
-      <IconButton
-        size="sm"
-        icon={FileUp}
+      <Button
+        // size="sm"
```
Comment (carlbarrdahl) [inline src/features/ballot/components/ImportCSV.tsx:49]: can remove this

File: src/features/ballot/components/ImportCSV.tsx [hunk 2]
```diff
@@ -52,6 +67,8 @@ export function ImportCSV({
           reader.readAsText(file);
           reader.onload = () => importCSV(String(reader.result));
           reader.onerror = () => console.log(reader.error);
+          // Set value to empty string to clear the file input, otherwise it won't trigger onChange again with the same file
+          e.target.value = "";
```
Comment (carlbarrdahl) [inline src/features/ballot/components/ImportCSV.tsx:71]: Nice one! Didn't know about this

File: src/features/ballot/components/ImportCSV.tsx [hunk 3]
```diff
@@ -72,10 +89,16 @@ export function ImportCSV({
...
+              )
+                .then(() => {
+                  onImport(rows);
+                  setRows([]);
+                })
+                .catch((error: Error) => {
+                  toast.error("Error importing CSV", {
+                    description: error.message,
+                  });
+                });
```
Comment (carlbarrdahl) [inline src/features/ballot/components/ImportCSV.tsx:101]: :+1:

**Review by carlbarrdahl — APPROVED** [review]
Looks good! Just delete the unused copy.

**Pull request merged**